### ISSUE TITLE
InvalidArgumentException on unknown default avatar option

### ISSUE THREAD

**Issue by strk** [sunchaserinfo/wp-libravatar-replace#1]
Enabling the plugin makes the wordpress instance unusable.
Logs shows fatal errors:

```
Fatal error:  Uncaught exception 'InvalidArgumentException' with message 'Invalid default avatar URL' in
/.../wp-content/plugins/libravatar-replace/classes/ServicesLibravatar.class.php:571\n

Stack trace:\n#0
/.../wp-content/plugins/libravatar-replace/classes/ServicesLibravatarExt.class.php(25): Services_Libravatar->processDefault('libravatar_defa...')\n#1
/.../wp-content/plugins/libravatar-replace/classes/ServicesLibravatar.class.php(177): ServicesLibravatarExt->processDefault('libravatar_defa...')\n#2
/.../wp-content/plugins/libravatar-replace/classes/LibravatarReplace.class.php(139): Services_Libravatar->getUrl('strk@xxx.io', Array)\n#3
/.../wp-content/plugins/libravatar-replace/classes/LibravatarReplace.class.php(122): LibravatarReplace->getAvatarCode(Object(ServicesLibravatarExt), 'strk@kbt.io', '', Array, 64)\n#4 [internal function]: LibravatarReplace->filterGetAvatar in
/.../wp-content/plugins/libravatar-replace/classes/ServicesLibravatar.class.php on line 571,
referer: .../wp-admin/update.php?action=install-plugin&plugin=libravatar-replace&_wpnonce=3897a9a7f2
```

The older "libravatar" plugin worked fine.

This is against Wordpress 4.5.2

**Comment by sunchaserinfo** [comment]
What is your default avatar setting?

**Comment by strk** [comment]
While the old plugin is activa, it is "Libravatar Logo".
When the old plugin is disactivated, none of the radio buttons are checked.

**Comment by strk** [comment]
Tweaking the setting before installing libravatar-replace fixes
the issue (but it's worth fixing the robustness bug).

**Comment by sunchaserinfo** [comment]
yeah, libravatar sets libravatar_default which the plugin doesn't recognize

**Comment by sunchaserinfo** [comment]
Fixed and released as 3.2.1
